backend/app/main.py

```
@@ -1223,6 +1297,108 @@ async def logout_user(response: Response):
+
+    if smtp.get("mock_mode"):
+        logger.info("[REACHOUT - Mock Mode] Would send reachout email")
+        logger.info("To: %s", to_email)
+        logger.info("Subject: %s", subject)
+        logger.info("User ID: %s", user_id)
+        return ReachoutResponse(success=True, message="Message sent")
+
+    try:
+        await asyncio.to_thread(_send_html_email_smtp, smtp, to_email, subject, html_body, reply_to)
+        return ReachoutResponse(success=True, message="Message sent")
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)